medication statements/README.md

```diff
@@ -0,0 +1,363 @@
+| MedicationStatement / component \[@typeCode="COMP"] / ehrSupplyAuthorise                                           | Mapping for `ehrSupplyAuthorise` is detailed in the section below: [EhrSupplyAuthorise Mapping](#ehrsupplyauthorise-mapping)       |
+| MedicationStatement / component \[@typeCode="COMP"] / ehrSupplyDiscontinue \[@classCode="SPY"] \[@moodCode="PRQ"]  | Mapping for `ehrSupplyDiscontinue` is detailed in the section below: [EhrSupplyDiscontinue Mapping](#ehrsupplydiscontinue-mapping) |
+| MedicationStatement / pertinentInformation \[@typeCode]                                                            | fixed value = `"PERT"`                                                                                                             |
+| MedicationStatement / pertinentInformation / pertinentMedicationDosage \[@classCode] \[@moodCode]                  | fixed value `@classCode="SBADM"`, fixed value `@moodCode="RMD"`                                                                    |
+ | MedicationStatement / pertinentInformation / pertinentMedicationDosage / text                                      | `MedicationRequest.dosageInstruction[0].text`                                                                                      |
+| MedicationStatement / participant \[@typeCode] \[@contextControlCode]                                              | fixed value = `@typeCode="AUT"`, fixed value = `@tcontextControlCode="OP"`. There may be zero, one or many participants            |
+| MedicationStatement / participant / agentRef \[@classCode]                                                         | fixed value = `"AGNT"`                                                                                                             |
+| MedicationStatement / participant / agentRef / id \[@root]                                                         | See <sup>3</sup>                                                                                                                   |
+
+1. If `medicationRequest.dispenseRequest.validityPeriod.end` does not exist then this value is not populated
+2. When `medicationRequest.requester.agent.referenceElement.resourceType` = `"Practitioner"` and `medicationRequest.requester.onBehalfOf` is populated then value is set from the reference to the `requester.agent` or `medicationRequest.requester.onBehalfOf`</br> Otherwise, when only `medicationRequest.requester.agent` exists then value is set from the reference to `requester.agent.reference` (when `resourceType` = `Practicioner`) or is the reference to the organisation (when `resourceType` = `Organisation`)</br>  If there are no matches for the previous options and `medicationRequest.recorder.reference` exists and the reference is either a `Practitioner`,`PractitionerRole` or `organisation` then the reference is set from the reference to `medicationRequest.recorder.reference` 
```

Comment:
```suggestion
2. When `medicationRequest.requester.agent.referenceElement.resourceType` = `"Practitioner"` and `medicationRequest.requester.onBehalfOf` is populated then value is set from the reference to the `medicationRequest.requester.agent` or `medicationRequest.requester.onBehalfOf`</br> Otherwise, when only `medicationRequest.requester.agent` exists then value is set from the reference to `requester.agent.reference` (when `resourceType` = `Practicioner`) or is the reference to the organisation (when `resourceType` = `Organisation`)</br>  If there are no matches for the previous options and `medicationRequest.recorder.reference` exists and the reference is either a `Practitioner`,`PractitionerRole` or `organisation` then the reference is set from the reference to `medicationRequest.recorder.reference` 
```